launcher/README.md

```diff
@@ -1,24 +1,40 @@
-The preflight updater GUI currently supports both PyQt4 and PyQt5. To
-enforce the use of PyQt5, set the environment variable SDW_UPDATER_QT to 5.
+# securedrop-workstation updater
 
-## Why support PyQt4 and PyQt5?
+The Updater ensures that the SecureDrop Workstation is up-to-date by checking for and applying any necessary VM updates, which may prompt a reboot.
 
-Qubes 4.0.3 uses an end-of-life Fedora template in dom0 (fedora-25). See
-rationale here:
+## Running the Updater
 
-https://www.qubes-os.org/doc/supported-versions/#note-on-dom0-and-eol
+Qubes 4.0.3 uses an end-of-life Fedora template in dom0 (fedora-25). See rationale here: https://www.qubes-os.org/doc/supported-releases/#note-on-dom0-and-eol.
 
-fedora-25 only includes PyQt4, which is why we have to support it for now.
-The next version of Qubes, Qubes 4.1, will include PyQt5 in dom0.
+To run the preflight updater:
+1. Open a `dom0` terminal
+2. Run `/opt/securedrop/launcher/sdw-launcher.py --skip-delta 0`
 
-## Installing PyQt4
+To run the notifier that pops up if `/proc/uptime` (how long the system has been on since its last restart) is greater than 30 seconds and `~/.securedrop_launcher/sdw-last-updated` (how long it's been since the Updater last ran) is greater than 5 days:
+1. Open a `dom0` terminal
+2. Run `/opt/securedrop/launcher/sdw-notify.py --skip-delta 0`
```

Comment:
`sdw-notify.py` does not currently support the `--skip-delta` argument. We could add that, or perhaps a more straightforward `--force` or similar.

Reply:
yeah, not sure I want to propagate the term `skip-delta`. switching to `force` would be less confusing to developers. i'll remove it from the docs here to address this for now.